docs/src/examples/components/Avatar/Variations/AvatarExampleSrc.shorthand.tsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react'
 import { Avatar } from '@stardust-ui/react'
 
-const AvatarExampleSrcShorthand = () => <Avatar src="/public/images/avatar/small/matt.jpg" />
+const AvatarExampleSrcShorthand = () => <Avatar src="public/images/avatar/small/matt.jpg" />
```

Comment:
Not part of this PR, but just as something we should think about.  There is no shorthand for `src`, so this example is not applicable.  We could show shorthand by showing how the other components implement an `image` prop using the shorthand `src` value, but it might be a bit confusing.

Again, not for this PR but we can discuss this later.